source/grid/tria.cc

```diff
@@ -15998,6 +16045,21 @@ void Triangulation<dim, spacedim>::load_attached_data(
 }
 
 
+template <int dim, int spacedim>
+DEAL_II_CXX20_REQUIRES((concepts::is_valid_dim_spacedim<dim, spacedim>))
+void Triangulation<dim, spacedim>::update_cell_relations()
+{
+  // Reorganize memory for local_cell_relations.
+  this->local_cell_relations.clear();
+
+  this->local_cell_relations.reserve(this->n_global_active_cells());
+
+  for (const auto &cell : this->active_cell_iterators())
+    this->local_cell_relations.emplace_back(
+      cell, ::dealii::CellStatus::cell_will_persist);
```

Comment:
This does not look right. I guess for load/save it does not matter, however, once actually interpolation is performed than we need to set this correctly.

Reply:
I agree.

Note: this implementation was copied by `p::f::T`. This means that `SolutionTransfer` does not currently support interpolation between `p::f::T`s.